lib/libutee/tee_api_arith_mpi.c

```diff
@@ -164,7 +145,10 @@ TEE_Result TEE_BigIntConvertFromOctetString(TEE_BigInt *dest,
 	if (sign < 0)
 		mpi_dest.s = -1;
 
-	put_mpi(&mpi_dest);
+	if (!res)
+		res = copy_mpi_to_bigint(&mpi_dest, dest);
+
+	mbedtls_mpi_free(&mpi_dest);
```

Comment:
maybe prefer keeping a local function `put_mpi()` to balance the `get_mpi()` calls.

Reply:
too late. forget it...

Reply:
Yeah, I'm sorry, I think sometimes I should give a bit more time for review.

Reply:
It's ok, I'm often late at reviewing, so better not wait for my comments if you're happy with the changes.